web-ui/representation/step.ml

```diff
@@ -0,0 +1,17 @@
+type t = {
+  status : string;
+  created_at : string;
+  finished_at : string;
+  queued_for : string;
+  ran_for : string;
+}
```

Comment:
Maybe just use `[@@deriving yojson]` and avoid a hand written function.
Would be nice to have simple tests on this since it forms a kind of API.